CorsixTH/Src/th_gfx.cpp

```diff
@@ -1479,6 +1481,15 @@ void animation::depersist(lua_persist_reader* pReader) {
   pReader->set_error("Cannot depersist animation instance");
 }
 
+void animation::set_overlay(animation_overlay_flags flags) {
+  // Sanity Check
```

Comment:
Fail early, or don't do this check and just accept the flag. I don't like silently changing the input.